traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -679,6 +552,11 @@ func validateV4(s *tc.ServerV40, tx *sql.Tx) (string, error) {
 	if !serviceAddrV6Found && !serviceAddrV4Found {
 		errs = append(errs, errors.New("a server must have at least one service address"))
 	}
+
+	if s.UpdPending == nil && s.ConfigUpdateTime == nil {
+		errs = append(errs, errors.New("either 'updPending' or 'configUpdateTime' may be null, but not both"))
```

Comment:
Is this true? I'd think we wouldn't really care about these fields for creating or updating a server...

Reply:
Previously (er currently) `updPending` is a required field in [validateCommon()](https://github.com/apache/trafficcontrol/blob/7185a0220f26a82e7b7e8579d56527be6baf001b/traffic_ops/traffic_ops_golang/server/servers.go#L513)

I don't disagree. I don't necessarily think we should be setting revalPending or updPending (or the new timestamp fields) with these CRUD endpoints for servers. But I was trying to stay consistent with the current implementation while also trying to prevent conflicting logic. To prevent the conflicting logic, I pulled the `null` check from the `validateCommon()` function. But now you'll see the check elsewhere so as to keep the original consistency.

Reply:
Removed